tekton/images/koparse/koparse/test_release.yaml

```diff
@@ -255,7 +255,7 @@ metadata:
 ---
 apiVersion: v1
 data:
-  image: gcr.io/k8s-prow/entrypoint@sha256:7c7cd8906ce4982ffee326218e9fc75da2d4896d53cabc9833b9cc8d2d6b2b8f
+  image: ghcr.io/k8s-prow/entrypoint@sha256:7c7cd8906ce4982ffee326218e9fc75da2d4896d53cabc9833b9cc8d2d6b2b8f
```

Comment:
Should this be reverted? Not an image we control and doesn't appear to be available on GHCR

Reply:
Yes, I will revert that... it's just a test file for `koparse` though, so it could be `foo.bar` and still work :D